options/locale/locale_en-US.ini

```diff
@@ -1231,6 +1231,9 @@ projects.board.color = "Color"
 projects.open = Open
 projects.close = Close
 projects.board.assigned_to = Assigned to
+projects.card_type.desc = "Card Previews"
+projects.card_type.images_and_text = "Images and Text"
+projects.card_type.text_only = "Text Only"
```

Comment:
Project Type was split into two sub-types:

* Board Type
* Card Type

This was required because Board Type is for layout, Card Type is for individual items on the board.

I used Board/Card because "Items" is used everywhere and would be difficult to grep (less ambiguous), and Board/Card is used by Trello anyway.